app/models/grda_warehouse/project_group.rb

```diff
@@ -28,27 +44,43 @@ class ProjectGroup < GrdaWarehouseBase
     has_many :organization_contacts, through: :projects
 
     scope :viewable_by, ->(user) do
-      if user.can_edit_project_groups?
-        current_scope
-      elsif current_scope.present?
-        current_scope.merge(user.project_groups)
-      else
-        user.project_groups
-      end
+      editable_by(user)
```

Comment:
Include a todo for `:can_view_assigned_project_groups`? While it doesn't seem crucial, I think I have a case where a user should be able to look to see what projects are in a group, but not change it.

Reply:
Yeah, I think there could be utility in being able to see the makeup of the project groups that have been assigned for your use.